### ISSUE TITLE
Low Priority: Owls get stuck on rocks

### ISSUE THREAD

**Issue by MongoTheElder** [Darkosto/SevTech-Ages#193]
Owls tend to get stuck on rocks. After a time, you end up with a lot of rooted owls

![2017-11-20_18 13 02](https://user-images.githubusercontent.com/29290246/33046388-b8b35018-ce1e-11e7-98d1-fa3141cd2fdb.png)

![2017-11-20_18 13 16](https://user-images.githubusercontent.com/29290246/33046393-bc45b478-ce1e-11e7-8699-863e923103b1.png)

Alpha 8, Twitch build

**Comment by Darkosto** [comment]
Yep! Still hoping McJty will be able to take a look at this.

https://github.com/McJty/ImmersiveCraft/issues/35

**Comment by justinrusso** [comment]
Fixed in dev. Waiting on a new release

**Comment by Darkosto** [comment]
Looks to be much better than before, they still get stuck from time to time but it looks good to me.
